Dockerfile.build

```diff
@@ -1,10 +1,11 @@
 FROM golang:1.10
 RUN apt-get update && apt-get -y install iptables
 
-RUN go get github.com/tools/godep \
-		github.com/golang/lint/golint \
+RUN go get github.com/golang/lint/golint \
 		golang.org/x/tools/cmd/cover \
```

Comment:
go get is called one time with multiple arguments, so maybe can do the fetch in parallel or something, instead splitting it will force to have 1 call per argument. Would keep this as is, hoping that go get can do some magic :D